packages/synthetic-chain/src/lib/vat-status.js

```diff
@@ -62,6 +62,14 @@ const makeSwingstore = db => {
       if (!targetVat) throw Error(`vat not found: ${vatName}`);
       return targetVat;
     },
+    /** @param {string} string a string to search for within the vat name. */
```

Comment:
using `string` as the type, the param name, and the doc word ready slowly. consider,
```suggestion
    /** @param {string} substring to search for within the vat name. */
```